README.md

```diff
@@ -23,6 +23,19 @@ Once connectors are deemed mature enough, they may be promoted into the Debezium
 
 Please see the [README.md](https://github.com/debezium/debezium#building-debezium) in the main repository for general instructions on building Debezium from source (prerequisites, usage of Docker etc).
 
+### Building the Cassandra connector
+
+In order to build the Cassandra connector you'll need JDK <= 9 because Cassandra
```

Comment:
Wow, is this true indeed? Do you have a reference?

Reply:
LOL it didn't compile for me with JDK 11 and I had to download and use JDK 8 for building it. I think that source was a stackoverflow response.
Now that you were asking I found https://cassandra.apache.org/doc/latest/new/java11.html#support-matrix and that there's `ant -Duse.jdk=11`. I will try if that helps and document that instead.

Reply:
https://issues.apache.org/jira/browse/CASSANDRA-9608 introduces Java 11 compatibility with Cassandra 4.0. The connector currently uses Cassandra 3.x

Reply:
Done